modules/understanding-upgrade-channels.adoc

```diff
@@ -7,79 +7,89 @@
+
+* candidate-{product-version}
+* fast-{product-version}
+* stable-{product-version}
+
+RC release versions are only available in candidate-{product-version}.
+
+GA release versions are only availble in fast-{product-version} or stable-{product-version}.
+
+More details about each channel and their usage are in the following sections.
+
+= candidate-{product-version}
+
+The candidate channel allows upgrading from an RC version to a later RC version and
+upgrading from a previous minor version of {product-title} to the current RC version.
+You cannot upgrade from an RC release to a GA release of {product-title}.
 
 = fast-{product-version}
 
-The fast channel is updated with new {product-version} patch versions as soon as Red
-Hat declares they are generally available. Use this channel if you wish to receive
-updates as soon as they are available or for your pre-production environments when
-participating in the connected customer program. This channel will contain all
-z-stream ({product-version}.z) updates but will not suggest upgrades to the next
-minor release (4.5.z) when the next minor release is available.
+The fast channel, containing only GA releases, is updated with new
+{product-version} patch versions as soon as Red Hat declares they are generally
+available. Use this channel if you wish to receive updates as soon as they are
+available or for your pre-production environments when participating in the connected
+customer program. This channel will contain all z-stream ({product-version}.z)
+updates but will not suggest upgrades to the next minor release (4.5.z) when the next
+minor release is available. This channel also includes the upgrade from the previous
+4.3 release.
 
 = stable-{product-version}
 
-The stable channel will contain updates on a time delay as they are gradually
-rolled out to customers based on data from our SRE teams, support services, and
-pre-production and production environments that participate in our connected
-customer program, rather than being immediately available as they are in the fast
-channel. For patch and CVE fixes this can range from several hours to a day and
-allows an extra period of assessment in how the software performs. If issues are
-detected during rollout, upgrades to that version may be blocked in both the fast
-and stable channels, and a new version may be introduced that will be the new
-preferred upgrade target.
+The stable channel, containing only GA releases, will contain updates on a time delay
+as they are gradually rolled out to customers based on data from our SRE teams,
+support services, and pre-production and production environments that participate in
+our connected customer program, rather than being immediately available as they are
+in the fast channel. For patch and CVE fixes this can range from several hours to a
+day and allows an extra period of assessment in how the software performs. This
+channel also includes the upgrade from the previous 4.3 release.
```

Comment:
Rewording ideas; take anything that sounds useful:

> The stable-{product-version} channel is the {product-version} channel to which releases are promoted after they have been promoted to fast-{product-version} [would be nice to link to the fast section here]. Membership in stable-{product-version} is, like membership in fast-{product-version}, a declaration that the release is supported. Releases will enter stable-{product-version} with a delay after they are promoted to fast-{product-version}.  For patch and CVE fixes, the delay can range from several hours to a day and allows an extra period of assessment in how the software performs.

Reply:
For customers and business `fast` and `stable` gets the releases at the same time. Technically we should not differentiate between them with respect to publishing them in to respective channels. If we do not agree on this we should ask the release-admins for clarification.

Reply:
> Technically we should not differentiate between them with respect to timing of publishing them in to respective channels.

Maybe releases at the same time, but update recommendation edges are delayed in stable for phased rollouts.